packages/connect/src/types/device.ts

```diff
@@ -90,7 +90,7 @@ export type KnownDevice = BaseDevice & {
     label: string;
     error?: typeof undefined;
     firmware: DeviceFirmwareStatus;
-    firmwareRelease?: ReleaseInfo | null;
+    firmwareReleaseConfig?: FirmwareReleaseConfigInfo;
```

Comment:
this is potentially breaking but I don't think 3rd parties are supposed to use it

Reply:
Yes, here I preferred doing a clear change by changing the name of the attribute so we make clear it is something different than the previous one before.  But yes it is a breaking change so we could properly state in changelog and make a minor release next time.

Reply:
I think this is ok.